content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+
+{{< img src="real_user_monitoring/browser/troubleshooting/error_rum_internal_context.png" alt="Error get internal context command">}}
+
+You can also check your browser developer tools console or network tab if you notice any errors related to the loading of the Browser RUM SDK.
+
+### Check that data is sent to the Datadog intake
+
+The Browser RUM SDK sends batches of data periodically to the Datadog intake. You should see network requests targeting `/v1/input` (the URL origin part may differ due to RUM configuration) in the Network section of your browser Developer Tools:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/network_intake.png" alt="RUM requests to Datadog intake">}}
+
+## RUM Cookies
+
+The Browser RUM SDK relies on cookies to store session information and follow a [user session][7] across different pages. The cookies are 1st party (they are set on your domain) and are not used for cross-site tracking. Here are the cookies set by the Browser RUM SDK:
+
+| Cookie name        | Details                                                                                                                                                                                                                                                                                                  |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `_dd_s`             | Cookie used to group all events generated from a unique user session across multiple pages. It contains the current session id, whether the session is excluded due to sampling and the expiration date of the session. The cookie is extended for an extra 15 minutes every time the user interacts with the website, in the limit of the maximum user session duration of 4 hours.|
+| `dd_site_test_*`   | Temporary cookie used to test for cookie support. Expires instantly.                                                                                                                                                                                                                                     |
+| `dd_cookie_test_*` | Temporary cookie used to test for cookie support. Expires instantly.                                                                                                                                                                                                                                     |
+
+**Note**: The following cookies have been used in the past: `_dd_l`, `_dd_r` and `_dd`. They have since been replaced by `_dd_s` in recent versions of the SDK and had the same purpose.
```

Comment:
Do readers of the Datadog docs really need to know this? I suggest deleting.

Reply:
Unfortunately some users may still rely on these cookies because they're using older versions of the SDK. I would be in favor of keeping that information.